packages/mobile/src/screens/app-screen/AppScreen.tsx

```diff
@@ -28,6 +30,11 @@ export const AppScreen = () => {
           name='FeatureFlagOverride'
           component={FeatureFlagOverrideModalScreen}
         />
+        <Stack.Screen
+          name='ChangePassword'
+          component={ChangePasswordModalScreen}
+        />
+        <Stack.Screen name='ChangeEmail' component={ChangeEmailModalScreen} />
```

Comment:
FYI I imitated another multipage "modal" I found, `EditTrackModalScreen` - lmk if that's not the right play for this one

Reply:
that one is perfect for this usecase. that one is slightly extra over-engineered and complex, but for now this is a fine pattern